src/controller/CommissioningWindowOpener.h

```diff
@@ -100,7 +101,12 @@ class CommissioningWindowOpener
     CHIP_ERROR OpenCommissioningWindow(NodeId deviceId, System::Clock::Seconds16 timeout, uint32_t iteration,
                                        uint16_t discriminator, Optional<uint32_t> setupPIN, Optional<ByteSpan> salt,
                                        Callback::Callback<OnOpenCommissioningWindow> * callback, SetupPayload & payload,
-                                       bool readVIDPIDAttributes = false);
+                                       bool readVIDPIDAttributes = false
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+                                       ,
+                                       bool jointCommissioning = false
```

Comment:
I think having explicit APIs for "open joint commissioning window" might be better, in terms of readability.  Not least because then we don't have random booleans at callsites that are not clear in terms of what they mean.

Reply:
Addressed in latest commit